sos/report/plugins/iscsi.py

```diff
@@ -41,11 +41,18 @@ def postproc(self):
         # to
         #
         # node.session.auth.password = ********
-        nodesessionpwd = r"(node\.session\.auth\.password\s+=\s+)(\S+)"
-        discoverypwd = r"(discovery\.sendtargets\.auth\.password\s+=\s+)(\S+)"
+        nodesessionpwd = r"(node\.session\.auth\.password(_in)?\s+=\s+)(\S+)"
+        discoverypwd = (
+                r"(discovery\.sendtargets\.auth\.password(_in)!"
```

Comment:
Is the `!` actually required here? It seems somewhat out of place to me at first glance, but I also am not fully aware of the context of what we're targeting here.

Reply:
That really seems like a typo, the format of either `node.session` and `discovery.sendtargets` is the same.

Reply:
Yes, was a typo while I was trying to fix the line that had more than 80 chars, sorry! I'll force push the fix soon.